### PR TITLE
Add Nuget package name to default template (#321)

### PR DESCRIPTION
**Pull request by qinezh**
@superyyrrzz @vwxyzh  @hellosnow @ansyral @chenkennt

### CODE REVIEW

File: src/docfx.website.themes/default/partials/class.tmpl.partial [hunk 1]
```diff
@@ -18,6 +18,9 @@
 {{/inheritance.0}}
 <h6><strong>{{__global.namespace}}</strong>:{{namespace}}</h6>
 <h6><strong>{{__global.assembly}}</strong>:{{assemblies.0}}.dll</h6>
```
Comment (hellosnow) [inline src/docfx.website.themes/default/partials/class.tmpl.partial]: have one question: need we also add check for assemblies.0? Or it must exist?

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
also update iframe.html.zip and statictoc.zip

**Comment by qinezh** [comment]
@superyyrrzz Updated

**Comment by superyyrrzz** [comment]
:+1:

**Pull request merged**